### ISSUE TITLE
ServerCore's feasibility 

### ISSUE THREAD

**Issue by sam4215** [paulcash/Servercore#5]
I know that this plugin is a good idea, but that's where @paulcash ends. @TBotV63 has done 99% of the work and that's a bit unfair. Also, when the "docs" are hosted on your Google site, don't just copy-paste class files in there. AT LEAST use GH. Discussion VVVV

**Comment by sam4215** [comment]
Oh, kudos to @MrMysteri0us too.

**Comment by TBotV63** [comment]
I agree that it's a good idea. What bugs me is that there are lots of kind and helpful individuals that are doing nothing but trying to help him learn how to program in OOP languages (specifically Java) and are giving him examples. I provided hime with a rudimentary framework to build upon, all he has to do is read the sourcecode. If he doesn't understand something, I'd be glad to help him and so would a lot of other people.
